go.mod

```diff
@@ -42,6 +42,7 @@ replace (
 
 replace (
 	github.com/ease-lab/vhive/examples/protobuf/helloworld => ./examples/protobuf/helloworld
+	github.com/ease-lab/vhive/utils/benchmarking/eventing => ./utils/benchmarking/eventing
```

Comment:
Because it is required in [examples/protobuf/helloworld/helloworld.pb.go](https://github.com/ease-lab/vhive/blob/tracing-fixes/examples/protobuf/helloworld/helloworld.pb.go)

- imported [here](https://github.com/ease-lab/vhive/blob/tracing-fixes/examples/protobuf/helloworld/helloworld.pb.go#L24)
- used [here](https://github.com/ease-lab/vhive/blob/tracing-fixes/examples/protobuf/helloworld/helloworld.pb.go#L45)

Reply:
how did you specify this dependency? is it somehow defined in the protofile or you had to hack it?